CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # CHANGELOG
 
+### 3.6.0
+- Remove the "EPSG:2154" by default and replace it with a parameter, if it's not in the lidar file.
```

Comment:
c'est pas vraiment ça qui se passeje crois : c'est plutôt "if the parameter is empty, the spatial reference is read from the las header"

Reply:
Oui, j'ai mal tourné ma phrase